stacktracer/src/main/scala-2/zio/internal/stacktracer/Tracer.scala

```diff
@@ -28,6 +28,6 @@ object Tracer {
 sealed trait Tracer {
   type Type <: AnyRef
   val empty: Type with Tracer.Traced
-  def unapply(trace: Type): Option[(String, String, Int)]
+  def parseOrNull(trace: Type): (String, String, Int)
```

Comment:
Can we make this method package-private?

Reply:
Also since we're adding a new method, we should perhaps we should return a case class here to avoid the boxing of the `Int`

Reply:
Done